cli/tests/unit/flash_test.ts

```diff
@@ -1122,68 +1079,6 @@ Deno.test(
   },
 );
 
-Deno.test("upgradeHttpRaw tcp", async () => {
```

Comment:
This unstable API will be removed along with flash implementation. Effectively there's no way to test it and the API will always throw.